docs/manage-persistent-storage.md

```diff
@@ -0,0 +1,391 @@
+charmed-etcd/5  logs/8      filesystem  rootfs        /var/snap/charmed-etcd/common/var/log/etcd  76 GiB   attached  
+```
+
+## Different cluster scenario
+In this scenario, we want to reuse existing storage from another etcd cluster/database.
+
+### Safe removal
+>**Attention - Before you remove your etcd cluster:**
+> - Safe the credentials for the admin-user
+> - or configure a user-defined password
+
+By default, charmed etcd enables authentication. That means you can not access an existing etcd database without 
+providing credentials. This is also valid for the admin-user charmed etcd uses to operate the cluster.
+
+At any time before removing your existing etcd cluster, you can provide a user-defined password for the admin user and 
+configure it to charmed etcd. Please refer to [manage-passwords](./manage-passwords.md) or the following steps:
+
+Create a juju secret with your desired password and make note of the secret's URI:
+```shell
+juju add-secret mysecret root=changeme
+```
+
+Example output:
+```shell
+secret:cuvh9ggv7vbc46jefvjg
+```
+
+Allow the charmed etcd application to access this secret:
+```shell
+juju grant-secret mysecret charmed-etcd
+```
+
+Configure the secret's URI as `system-users` credentials to charmed etcd:
+```shell
+juju config charmed-etcd system-users=secret:cuvh9ggv7vbc46jefvjg
+```
+
+Now it is safe to remove your existing charmed etcd application:
+```shell
+juju remove-application charmed-etcd
+```
+
+Example output:
+```shell
+WARNING This command will perform the following actions:
+will remove application charmed-etcd
+- will remove unit charmed-etcd/3
+- will remove unit charmed-etcd/4
+- will remove unit charmed-etcd/5
+- will remove storage logs/6
+- will remove storage logs/7
+- will remove storage logs/8
+- will detach storage data/0
+- will detach storage data/2
+- will detach storage data/4
+
+Continue [y/N]? y
+```
+
+You can check the status after removal by running the `juju status --storage` command:
+```shell
+Model  Controller      Cloud/Region         Version  SLA          Timestamp
+etcd   dev-controller  localhost/localhost  3.6.0    unsupported  13:11:11Z
+
+Storage Unit  Storage ID  Type        Pool          Mountpoint  Size     Status    Message
+              data/0      filesystem  etcd-storage              8.0 GiB  detached  
+              data/2      filesystem  etcd-storage              8.0 GiB  detached  
+              data/4      filesystem  etcd-storage              8.0 GiB  detached  
+
+Model "admin/etcd" is empty.
+```
+
+### Deploy new cluster with existing storage
+After you removed your charmed etcd application, deploy a new one attaching the `data` volume with most recent data and 
+configure the secret URI containing the admin user's password:
+```shell
+juju deploy charmed-etcd etcd --attach-storage=data/0 --config system-users=secret:cuvh9ggv7vbc46jefvjg
+```
```

Comment:
Please add a similar note on why it is important to start with 1 unit, with the most up-to-date data (latest unit) only then scale up